src/views/dashboard/playground/docs/getting-started.md

```diff
@@ -16,23 +16,39 @@ from [GreptimeCloud](https://greptime.com/product/cloud). **You can also explore
 The instance is valid within **1 hour** once initiated, and you will need to create a new one when the time is up. So please never store important data in Greptime Play sessions.
 :::
 
+## Sample data
+
+We will focus on an example of CPU usage in this document. The example is based on a table named `cpu_metrics`, which contains the following columns:
+
+- `hostname`: the host name of the machine
+- `environment`: the environment of the service, e.g. production, staging, etc.
+- `usage_user`: the percentage of CPU utilization that occurred while executing at the user level (application)
+- `usage_system`: the percentage of CPU utilization that occurred while executing at the system level (kernel)
+- `usage_idle`: the percentage of time that the CPU or CPUs were idle and the system did not have an outstanding disk I/O request
+- `ts`: the timestamp of the record
+
+We will create a table named `cpu_metrics` and insert some data into it. Then we will use SQL to query the data. The protocol used in this example is SQL. Let's get started!
+
+
 ## Create a Time-Series Table
 
-Let's start the journey by creating a simple `system_metrics` table first. Note that we pre-defined `host`
-and `idc` as the primary keys; `ts` as time index, both are important to know
-in GreptimeDB. Click `Run` on the upper left in the panel below to create the table:
+GreptimeDB offers a schemaless approach to writing data that eliminates the need to manually create tables using additional protocols. See [Automatic Schema Generation](https://docs.greptime.com/user-guide/write-data#automatic-schema-generation). 
 
+Here we need to create a table manually with SQL protocol. 
+Let's start the journey by creating a simple `cpu_metrics` table. Note that we pre-defined 
+`hostname` as the primary key; `ts` as time index, both are important to know
```

Comment:
hostname and environment as primary key because we need both field as identifier of a host